src/board/sensors.ts

```diff
@@ -123,3 +123,38 @@ export class EnumSensor extends Sensor {
     };
   }
 }
+
+export interface LogData {
+  key: string;
+  value: string | number;
+}
+
+export class DataLoggingSensor extends Sensor {
```

Comment:
Similarly to radio, I'm unsure whether this fits the sensor model. Let's see how radio works out first.